packages/expo-print/ios/ExpoPrintToFile.swift

```diff
@@ -0,0 +1,103 @@
+import ExpoModulesCore
+
+public class ExpoPrintToFile {
+  var renderTasks: [ExpoWKPDFRenderer] = []
+  let appContext: AppContext?
```

Comment:
```suggestion
  weak let appContext: AppContext?
```
The current version shouldn't leak, but to be safe, it's better to make appContext a weak reference.